### PR TITLE
Feature/clean up styling

### PR DESCRIPTION
**Pull request by hoomberto**
# Description

Added styling and formatting to reflect the comp provided for project - adjusted spacing between sections, formatted font colors, sizes, and styles, and added hidden HTML tags for future iterations. 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Load index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,53 +4,68 @@
...
+            <section class="intention-container">
+                <h3>What would you like to accomplish during this time?</h3>
+                <input class="user-input" id="intentions" type="text"></input>
+                <p class="error hidden"><img class="error-icon" src="./assets/warning.svg">A description is required.</p>
+            </section>
+            <section class="minutes-seconds-section">
+              <div class="minutes-container">
+                <label for="minutes">Minutes</label>
+                <input id="minutes" type="text"></input>
+                <p class="error hidden"><img class="error-icon" src="./assets/warning.svg">A description is required.</p>
```
Comment (AshleyOh-bit) [inline index.html:47]: Clever to add the image tag inside the p class for formatting.

File: styles.css [hunk 2]
```diff
@@ -85,10 +124,15 @@ button {
 
 h1 {
   color: #CBC9CF;
+  text-align: center;
+  text-transform: uppercase;
```
Comment (tylrs) [inline styles.css:128]: Text-transform: uppercase what a neat trick

File: styles.css [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
-    src: url(https://fonts.googleapis.com/css2?family=Montserrat&display=swap);
+html {
+    font-family: 'Montserrat', sans-serif;
   }
 
 body {
   background-color: #46424D;
+  margin: 0;
+  height: auto;
+  overflow-y: hidden;
```
Comment (tylrs) [inline styles.css:14]: Did not know about the overflow-y property

File: styles.css [hunk 4]
```diff
@@ -3,15 +3,15 @@
   color: #FFF;
 }
 
-@font-face {
-    font-family: Monttserrat;
```
Comment (AshleyOh-bit) [inline styles.css:7]: Curious as to why this global font-face selector wasn't the preferred method? Glad you found a more concise way of implementing the font-family, though.

**Review by tylrs — COMMENTED** [review]
We could still update the grid percentages

**Review by AshleyOh-bit — APPROVED** [review]

**Pull request merged**